src/content/docs/apis/nerdgraph/examples/mobile-monitoring-config-nerdgraph.mdx

```diff
@@ -100,67 +99,56 @@ Variables:
 }
 ```
 
-For more information on what mobile settings can be updated via NerdGraph, reference the following mutation. Documentation for each field can be found in the NerdGraph explorer.
+## Configure network request settings
+
+Here's an example of how mobile settings can be updated via NerdGraph:
```

Comment:
@CherylFrankenfield @asasha Can you review this sample and make sure it captures the ability to configure all three network request settings (block/allow hostnames, create aliases, ignore status codes)?

Reply:
Since no one is available to review this sample, I removed this section entirely.